.github/workflows/pr-tests-helm.yml

```diff
@@ -0,0 +1,116 @@
+name: Helm PR
+
+on:
+  workflow_call:
+    inputs:
+      charts:
+        required: true
+        type: string
+      charts_dir:
+        required: false
+        type: string
+        default: "charts"
+      helm_version:
+        required: false
+        type: string
+        default: "3.2.1"
+      k8s_version:
+        required: false
+        type: string
+        default: "1.24.0"
+
+jobs:
+  helm-lint:
+    name: helm lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        dir: ${{fromJson(inputs.charts)}}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Helm
+        uses: azure/setup-helm@20d2b4f98d41febe2bbca46408499dbb535b6258 # v3.0
```

Comment:
Could you add Renovate tags to all dependency version constraints, i.e. `# renovate: tag=v3.0`?

Reply:
Sure